client/me/help/style.scss

```diff
@@ -46,3 +46,38 @@
 	font-size: 14px;
 	color: $gray-dark;
 }
+
+.help__course-teaser .card {
```

Comment:
It'd be great to turn this into a prop of the Card component: `Card highlighted` or `Card primary` which would output a class `.card.is-highlighted` or `.card.is-primary`.